### ISSUE TITLE
lr scheduler `MultiStepLR` doesn't follow PyTorch's LRScheduler API.

### ISSUE THREAD

**Issue by ChirsHuanhuan** [Megvii-BaseDetection/BEVDepth#184]
when training I meet the error: pytorch_lightning.utilities.exceptions.MisconfigurationException: The provided lr scheduler `MultiStepLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.
Are there anyone who has met the problem too? How to solve it?

**Comment by YHCks** [comment]
同问
